keyboards/handwired/practice60/config.h

```diff
@@ -40,6 +40,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* define if matrix has ghost */
 //#define MATRIX_HAS_GHOST
 
+#define RGBLIGHT_STM32_SPI
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 8
+#define RGB_DI_PIN B15
+#define WS2812_SPI SPID2
+#define RGBLIGHT_SPI_DIVISOR (SPI_CR1_BR_1|SPI_CR1_BR_0)
```

Comment:
Just to clarify, the `WS2812_SPI` and `RGBLIGHT_SPI_DIVISOR` variables shouldn't be necessary in the config :)